CHANGELOG.md

```diff
@@ -1,5 +1,74 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.4.1
+
+<!-- generated comparing v5.4.0..master -->
+
+_Feb 7, 2022_
+
+A big thanks to the 24 contributors who made this release possible. Here are some highlights ✨:
+
+- ♿️ Ensured Snackbar messages are announced by screen reader (#30774) @eps1lon
```

Comment:
```suggestion
- ♿️ Snackbar messages are now announced by screen readers (#30774) @eps1lon
```

Reply:
It sounds like they weren't previously with your suggestion. 

We just fixed it for nvda+firefox. The other combinations were working as far as we know.

Though I don't know how to best describe this without making it sound boring 😆

Reply:
Oh, sorry. Thanks for the heads up! Updated it to be more accurate, let me know if it's correct.